docker/keeper-ksm/Pipfile

```
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+keeper-secrets-manager-core = "*"
+keeper-secrets-manager-helper = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
```

Are you sure you want installing older version?
```suggestion
python_version = "3.10"
```